infra/jpa/META-INF/spring/projectName-infra.xml

```diff
@@ -24,7 +24,7 @@
         <property name="jpaVendorAdapter" ref="jpaVendorAdapter" />
         <property name="jpaPropertyMap">
             <util:map>
-                <entry key="hibernate.hbm2ddl.auto" value="none" />
+                <entry key="hibernate.hbm2ddl.auto" value="" />

```

Comment:
I think that `validate` is good choice. However, it does not work if all of following conditions are matched. Because it is perform schema validation before schema initialization.

* If use the in-memory database (such as H2)
* If use the `<jdbc:initialize-database>` as schema initialization method

If this behavior want to change, we needs to change bean definitions as follows (one solution of several solutions):

* Define bean of `DataSourceInitializer` by manually to assign bean id
* Add the `depend-on` for `DataSourceInitializer` in bean definition of  `LocalContainerEntityManagerFactoryBean` to guarantee bean initialization order.

e.g)
```xml
<bean id="dataSourceInitializer" class="org.springframework.jdbc.datasource.init.DataSourceInitializer">
    <!-- ... -->
</bean>

<bean
    class="org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean"
    id="entityManagerFactory" depends-on="dataSourceInitializer">
    <!-- ... -->
</bean>
```

Which choice is better ?

* `<entry key="hibernate.hbm2ddl.auto" value="" />` (PR's solution)
* `<!-- <entry key="hibernate.hbm2ddl.auto" value="validate" /> -->` (comment out) + supplementary comment
* `<entry key="hibernate.hbm2ddl.auto" value="validate" />` +  supplementary comment
* others...